src/core/CMakeLists.txt

```diff
@@ -791,6 +792,7 @@ target_link_libraries(inviwo-core
         $<$<BOOL:${WIN32}>:inviwo::stackwalker>
         $<$<BOOL:${APPLE}>:${CORESERVICES_LIBRARY}>
         inviwo::tinydir
+        roaring
```

Comment:
you should link roaring::roaring

Reply:
we should also make it private now that we have hidden the headers...

Reply:
had to add an alias for `roaring::roaring`

Reply:
no, it should be there according the the docs?

Reply:
But it isn't for some reason...